### PR TITLE
Add instructions to run app locally to README

### PR DESCRIPTION
**Pull request by gemmai95**
Update README to include instructions for running service locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,19 @@ pipenv install
 ```
 
 ## Running
+
 [Install Docker](https://docs.docker.com/engine/installation/)
 ```bash
 docker-compose up
 ```
 
+or run locally (this requires postgres running locally on port 5432):
```
Comment (andrewmil) [inline README.md:31]: From some testing locally you also need to change the `ras-party-db.uri` to be `"postgres://postgres:postgres@localhost:5432/postgres"`

Comment (andrewmil) [reply README.md:31]: Or that should just be the new default maybe?

Comment (gemmai95) [reply README.md:31]: I changed it in the config file to localhost locally. I discussed this with @davidmort and he wanted to leave it with him for now.

**Review by andrewmil — APPROVED** [review]
Looks good if we know what we're doing about the database uri

**Review by tej99 — APPROVED** [review]

**Pull request merged**